src/app/domainregister/domainregister.component.ts

```diff
@@ -629,7 +629,7 @@ export class DomainRegisterComponent implements AfterViewInit {
   //  .timeout(60000) 
   //  .subscribe(
   //      data => {
-  //          var reply = data.json()
+  //          var reply = data
```

Comment:
Looks weird to have changes in commented out code :P Perhaps this should just be deleted if it's not used?